src/impl_/trampoline.rs

```diff
@@ -201,3 +219,30 @@ where
     py_err.restore(py);
     R::ERR_VALUE
 }
+
+/// Implementation of trampoline for functions which can't return an error.
+///
+/// Panics during execution are trapped so that they don't propagate through any
+/// outer FFI boundary.
+///
+/// Exceptions produced are sent to `sys.unraisablehook`.
+///
+/// # Safety
+///
+/// ctx must be either a valid ffi::PyObject or NULL
+unsafe fn trampoline_inner_void<F>(body: F, ctx: *mut ffi::PyObject)
```

Comment:
I initially stumbled while reading this and I think I would prefer if the name would reference "unraisable", e.g. "trampoline_inner_unraisable" instead of "void".